content/millennium/r4/proprietary-codes.md

```diff
@@ -168,6 +175,18 @@ This code set may include Marital Statuses such as Divorced and Married.
       "userSelected": true
     }
 
+#### Code Set 40 Person Relationship
+
+This code set describes the Relationship of one person to another.
```

Comment:
Need Kathy/Andy/Alex's suggestion on this description.

Reply:
Alex said he found this on the wiki 
"This code set is used to describe the patient's relationship to Subscriber, Guarantor, Emergency Contact, Next of Kin, and other related persons."

Reply:
https://github.com/cerner/fhir.cerner.com/pull/254/commits/56622c4899ac4a5fc33143311dd4cab6c10a7497#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R182

Reply:
https://github.com/cerner/fhir.cerner.com/pull/254/commits/56622c4899ac4a5fc33143311dd4cab6c10a7497#diff-f1a7812a3d6e9f4049fb6005eecfb1f5R180